test/.eslintrc

```diff
@@ -0,0 +1,3 @@
+env:
```

Comment:
Mocha tests use things like `describe` without being declared. Setting eslint environment to node, mocha only for the `/test` dir address this.